core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -111,14 +111,15 @@ oppia.controller('StatisticsTab', [
         // ending (numCompletions/numVisits), should we do this for all
         // terminal nodes, instead? If so, explorationStatesService needs to be
         // able to provide whether given states are terminal
-        for (var stateName in explorationStatesService.getStates()) {
+
+        explorationStatesService.getStateNames().map(function(stateName) {
```

Comment:
I spent some time deciding whether to use map or forEach. The first link I came across in my research just said that map is faster so I just went with map. After looking into it more, it seems like forEach may be the better choice as we don't need to collect the returned array from map and it's more readable. If there's any optimization in speed it's probably too small to make a real difference. Will change it to forEach